addons.yaml

```diff
@@ -156,3 +156,10 @@ microk8s-addons:
       check_status: "deployment.apps/sosivio-dashboard"
       supported_architectures:
         - amd64
+
+    - name: "ondat"
+      description: "Ondat is a software-defined, cloud native storage platform for Kubernetes."
+      version: "latest"
```

Comment:
Is the `latest` here correct? It seems the manifest included in this PR deploys a very specific version of the storagecli. Wouldn't there be an issue (after a year for example) where the latest would not be compatible with the manifest in this PR and/or the Kubernetes version?

Reply:
Yes the `latest` tag is correct here - we are also in the progress of creating a new `stable` tag for the storageoscli image that will always pull down the latest stable version whenever it is available (which is when we release a new version of ondat) - we plan to release this new tag soon.